hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -99,6 +102,19 @@ public static String offsetsToStr(OffsetRange[] ranges) {
       return sb.toString();
     }
 
+    /**
+     * Stringfy the offset ranges, used for logging
+     * Format: topic,0:fromOffset0->toOffset0,1:fromOffset1->toOffset1,...
+     */
+    public static String offsetsStringfy(OffsetRange[] ranges) {
```

Comment:
Instead of this method, can we simply use Arrays.toString(ranges) in the caller.